### DISCUSSION TITLE
[Bug] Issue with opacity modifier syntax.

### DISCUSSION THREAD

**Discussion by Bug-Finderr** [tailwindlabs/tailwindcss#15447] [Help]
## Details

**What version of Tailwind CSS are you using?**
v3.4.16

**What build tool (or framework if it abstracts the build tool) are you using?**
Vite 6, Next-js 15 (tried both)

**What version of Node.js are you using?**
v22.11.0

**What browser are you using?**
Chrome

**What operating system are you using?**
Windows

## **Reproduction URL**
[Github Link](https://github.com/Bug-Finderr/tailwind-bug)
NOTE: The main part is from [line 52 in src/app/page.tsx](https://github.com/Bug-Finderr/tailwind-bug/blob/d117df482f19dca6aa1b69210f3f7cccc772334d/src/App.tsx#L52)

## **Describe your issue**

I am **NOT SURE** if this is an actual issue with CSS or my skill issue. Raised this issue as I could not find an exact solution to this problem.

![image](https://github.com/user-attachments/assets/0c1206fa-8061-4064-9c56-23eb890cece0)
Notice the white artifacts around the border radius when [using opacity modifier syntax](https://tailwindcss.com/docs/text-color#changing-the-opacity).

For more details, visit the [repo](https://github.com/Bug-Finderr/tailwind-bug) or this [deployed link](https://bug-finderr.github.io/tailwind-bug/)

**Comment by philipp-spiess** [comment]
Hey there! Thanks for the writeup and definitely agree that this is odd behavior but the classes that Tailwind CSS generates look correct so this seems to be, as you mentioned, a browser rendering glitch:

<img width="249" alt="Screenshot 2024-12-19 at 12 08 59" src="https://github.com/user-attachments/assets/ecf844f2-d577-47ed-81a7-c6077213a87e" />

**Reply by Bug-Finderr** [reply]
Any advice on where exactly I should mention this problem for a possible fix?
